src/test/java/com/scottlogic/sorts/SorterComboTest.java

```diff
@@ -0,0 +1,91 @@
+            OffsetDateTime.of(2020, 3, 12, 13, 22, 12, 0, ZoneOffset.UTC),
+            "Another example post.", 3);
+
+    @Test
+    public void sort_validNameAndDate_sortedFirstByNameThenDate() {
+        List<UserPost> userPosts = Arrays.asList(userPost1, userPost2, userPost3);
+        AuthorPostSorter sortByAuthor = new AuthorPostSorter();
+        DatePostSorter sortByDate = new DatePostSorter();
+        SorterCombo sortByBoth = new SorterCombo(sortByAuthor, sortByDate);
+        List<UserPost> actualResult = sortByBoth.sort(userPosts, SortOrder.ASC);
+        List<UserPost> expectedResult = Arrays.asList(userPost3, userPost2, userPost1);
+        Assert.assertEquals(expectedResult, actualResult);
+    }
+    @Test
+    public void sort_validInput_sortedFirstByContentThenLikes() {
+        List<UserPost> userPosts = Arrays.asList(userPost1, userPost2, userPost3);
+        ContentPostSorter sortByContent = new ContentPostSorter();
+        LikesPostSorter sortByLikes = new LikesPostSorter();
+        SorterCombo sortByBoth = new SorterCombo(sortByContent, sortByLikes);
+        List<UserPost> actualResult = sortByBoth.sort(userPosts, SortOrder.ASC);
+        List<UserPost> expectedResult = Arrays.asList(userPost1, userPost3, userPost2);
+        Assert.assertEquals(expectedResult, actualResult);
+    }
+
+    @Test
+    public void sort_nullPostNameAndContent_NullIgnored() {
+        List<UserPost> userPosts = Arrays.asList(userPost1, userPost2, null, userPost3);
+        AuthorPostSorter sortByAuthor = new AuthorPostSorter();
+        ContentPostSorter sortByContent = new ContentPostSorter();
+        SorterCombo sortByBoth = new SorterCombo(sortByAuthor, sortByContent);
+        List<UserPost> actualResult = sortByBoth.sort(userPosts, SortOrder.DESC);
+        List<UserPost> expectedResult = Arrays.asList(userPost2, userPost1, userPost3);
+        Assert.assertEquals(expectedResult, actualResult);
+    }
+
+    @Test
+    public void sort_emptyList_emptyListReturned() {
+        List<UserPost> userPosts = new ArrayList<>();
+        AuthorPostSorter sortByAuthor = new AuthorPostSorter();
+        ContentPostSorter sortByContent = new ContentPostSorter();
+        SorterCombo sortByBoth = new SorterCombo(sortByAuthor, sortByContent);
+        List<UserPost> actualResult = sortByBoth.sort(userPosts, SortOrder.DESC);
+        Assert.assertTrue(actualResult.isEmpty());
+    }
+
+    @Test
+    public void sort_nullList_emptyListReturned() {
+        List<UserPost> userPosts = null;
+        LikesPostSorter sortByLikes = new LikesPostSorter();
+        ContentPostSorter sortByContent = new ContentPostSorter();
+        SorterCombo sortByBoth = new SorterCombo(sortByLikes, sortByContent);
+        List<UserPost> actualResult = sortByBoth.sort(userPosts, SortOrder.DESC);
+        Assert.assertTrue(actualResult.isEmpty());
+    }
+
+    @Test
+    public void sort_validInput_inputListNotMutated() {
```

Comment:
Nice work including this. It would be a good case to have in the AND and OR tests as well.